.travis.yml

```diff
@@ -55,5 +55,8 @@ script:
   - node_modules/.bin/gulp test/node
   - node_modules/.bin/gulp test/node -no-patch-clock
   - cp ./test/browser/custom-element.spec.js ./build/test/browser
+  - yarn tsc:es6
```

Comment:
please rename because es6 is not a thing, es2015 is the official name so let's be consistent :)

Reply:
Sure, got it!